Description: This patch fixes the pod2man error "Unknown directive: =HEAD1",
 reported by lintian.
Origin: vendor
Forwarded: https://rt.cpan.org/Ticket/Display.html?id=71252
Author: Alessandro Ghedini <al3xbio@gmail.com>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2011-09-26

--- a/FCGI.PL
+++ b/FCGI.PL
@@ -566,7 +566,7 @@
 
 =back
 
-=HEAD1 LIMITATIONS
+=head1 LIMITATIONS
 
 FCGI.pm isn't Unicode aware, only characters within the range 0x00-0xFF are 
 supported. Attempts to output strings containing characters above 0xFF results
